assets/constants.ts

```diff
@@ -6674,6 +6677,64 @@ export default {
         }
       ]
     },
+    "BLOG_HOMEPAGE": {
+      "ROUTE": "blog",
+      "TITLE": "I18N_BLOG_HOME_PAGE_TITLE",
+      "META": [
+        {
+          "PROPERTY_TYPE": "itemprop",
+          "PROPERTY_VALUE": "description",
+          // eslint-disable-next-line max-len
+          "CONTENT": "Read the latest on what's new and exciting with Oppia."
+        },
+        {
+          "PROPERTY_TYPE": "itemprop",
+          "PROPERTY_VALUE": "og:description",
+          // eslint-disable-next-line max-len
+          "CONTENT": "Read the latest on what's new and exciting with Oppia."
+        }
+      ]
+    },
+    "BLOG_HOMEPAGE_SEARCH": {
+      "ROUTE": "blog/search/find",
+      "TITLE": "I18N_BLOG_HOME_PAGE_TITLE",
+      "META": [
+        {
+          "PROPERTY_TYPE": "itemprop",
+          "PROPERTY_VALUE": "description",
+          // eslint-disable-next-line max-len
+          "CONTENT": "Read the latest on what's new and exciting with Oppia."
+        },
+        {
+          "PROPERTY_TYPE": "itemprop",
+          "PROPERTY_VALUE": "og:description",
+          // eslint-disable-next-line max-len
+          "CONTENT": "Read the latest on what's new and exciting with Oppia."
+        }
+      ]
+    },
+    "BLOG_POST_PAGE": {
+      "ROUTE": "blog/:blog_post_url_fragment",
+      "TITLE": "I18N_BLOG_POST_PAGE_TITLE",
+      // Some routes contain url fragments, as syntax for url fragments are
+      // different for angular router and backend. They have to be registered
+      // manually in the backend. Please use angular router syntax here.
```

Comment:
I'm a bit confused .. who is this comment being addressed to? ("Please use ...")

Reply:
The syntax of route in backend and here are different. So for developers to not get confused, and in case any change in route is made, angular route synatax shuld be used in constants.ts and backend syntax in main.py.

Reply:
Ah, I still think this is confusing, but didn't realize this was already repeated elsewhere in the file. In that case, let's leave it for now, thanks!